assembly/net/index.ts

```diff
@@ -157,13 +281,24 @@ for (let i = <usize>0; i < <usize>TCP_READ_BUFFER_COUNT; i++) {
   tcpReadVecsTemp += offsetof<iovec>();
 }
 
+
+  /**
+   * A TCP Socket that allows for reading and writing data from and to the TCP connection
+   * associated. This object is constructed by an associated `TCPServer` or the
+   * `TCPSocket.connect()` method. Typically, lunatic end users should not construct
+   * TCPSocket references directly using the  `new` keyword.
+   *
+   * @param {u32} socket_id - The host's socket_id.
+   */
 export class TCPSocket {
```

Comment:
I think that this should be a `TCPStream`. A socket is closer to the `TCPServer`, not a single connection stream.

Reply:
Yes sir! I thought that too at some point. Glad to see we agree.